### PR TITLE
Update objects_and_object_constructors.md

### PR DESCRIPTION
**Pull request by ariffdev**
## Because
The example code's comment states that the output of the expression should be Object {...} however, when tested it displays the constructor name. So: Player {...}

## This PR
Changes the Object in the output to Player

## Pull Request Requirements

-   [x] I have thoroughly read and understand [The Odin Project Contributing Guide](https://github.com/TheOdinProject/.github/blob/main/CONTRIBUTING.md)
-   [x] The title of this PR follows the `location of change: brief description of change` format, e.g. `Intro to HTML and CSS lesson: Fix link text`
-   [x] The `Because` section summarizes the reason for this PR
-   [x] The `This PR` section has a bullet point list describing the changes in this PR
-   [x] If this PR addresses an open issue, it is linked in the `Issue` section
-   [x] If any lesson files are included in this PR, they have been previewed with the [Markdown preview tool](https://www.theodinproject.com/lessons/preview) to ensure it is formatted correctly
-   [x] If any lesson files are included in this PR, they follow the [Layout Style Guide](https://github.com/TheOdinProject/curriculum/blob/main/LAYOUT_STYLE_GUIDE.md)

### CODE REVIEW

**Comment by thatblindgeye** [comment]
@ariffdev would you mind resolving the merge conflicts

**Comment by ariffdev** [comment]
> @ariffdev would you mind resolving the merge conflicts

@thatblindgeye done. Thanks for the information

File: javascript/organizing_your_javascript_code/objects_and_object_constructors.md [hunk 1]
```diff
@@ -233,7 +233,7 @@ Object.getPrototypeOf(Player.prototype) === Object.prototype; // true
 
 // Output may slightly differ based on the browser
 player1.valueOf(); // Output: Object { name: "steve", marker: "X", sayName: sayName() }
-~~~
+
```
Comment (thatblindgeye) [inline javascript/organizing_your_javascript_code/objects_and_object_constructors.md:236]: Looks like there was an issue from the merge conflicts. We need to add back in the tilde `~~~` here as well as your original update

Comment (ariffdev) [reply javascript/organizing_your_javascript_code/objects_and_object_constructors.md]: Just saw that. Thanks!

Comment (thatblindgeye) [reply javascript/organizing_your_javascript_code/objects_and_object_constructors.md]: Did you still want to update the `Output: Object` line to `Output: Player`?

Comment (ariffdev) [reply javascript/organizing_your_javascript_code/objects_and_object_constructors.md]: Either way is good
I think the comment that says that the output may differ is good enough

Comment (thatblindgeye) [reply javascript/organizing_your_javascript_code/objects_and_object_constructors.md]: In that case I will close this PR out

**Pull request closed without merging**